<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGitRepoViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="6">
     <widget class="QTableView" name="commit_list">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="hash_label">
      <property name="text">
       <string>Commit hash: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="filter_criteria">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Short log</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Author</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Date</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QGitRepoViewer::CSearchLineWidget" name="commit_search" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="branch_list">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The list of branches in git repository&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <item>
       <property name="text">
        <string>text</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>text2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="filter_label">
      <property name="text">
       <string>Search in: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="branch_label">
      <property name="text">
       <string>Repository branch: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="4">
     <widget class="QLineEdit" name="commit_hash">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_repo"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open_repo">
   <property name="text">
    <string>Open repository...</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGitRepoViewer::CSearchLineWidget</class>
   <extends>QWidget</extends>
   <header>CSearchLineWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>branch_list</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>CMainWindow</receiver>
   <slot>aboutBranchSelected(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter_criteria</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CMainWindow</receiver>
   <slot>aboutFilterChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aboutBranchSelected(QString)</slot>
  <slot>aboutFilterChanged(int)</slot>
 </slots>
</ui>
